orchestrate-containers/Ansible Scripts/App_deployment.yml

```diff
@@ -0,0 +1,103 @@
+---
+- hosts: master
+  remote_user: root
+
+  vars_prompt:
+    - name: "MariaPass"
+      prompt: "Please enter the password for MariaDB"
+      private: yes
+      confirm: yes
+
+    - name: "WordPass"
+      prompt: "Please enter the password for WordPress"
+      private: yes
+      confirm: yes
+
+    - name: "Prometheus"
+      prompt: "Please enter the Prometheus url"
+      private: no
+
+    - name: "Grafana"
+      prompt: "Please enter the Grafana url"
+      private: no
+
+    - name: "Alertmanager"
+      prompt: "Please enter the Alertmanager url"
+      private: no
+
+    - name: "Kibana"
+      prompt: "Please enter the Kibana url"
+      private: no
+
+  tasks:
+    - name: Creating Kubernetes directory
+      file: path=/kubernetes state=directory
+
+    - name: Download and unzip the K8s file from Github
+      unarchive:
+        src: https://github.com/CloudCommandos/missions/raw/CC/orchestrate-containers/k8s%20files/testApp.tar
+        dest: /kubernetes
+        remote_src: yes
+
+    - name: Download and unzip the Prometheus file from Github
+      unarchive:
+        src: https://github.com/CloudCommandos/missions/raw/CC/orchestrate-containers/k8s%20files/prometheus.tar
+        dest: /kubernetes
+        remote_src: yes
+
+    - name: Download and unzip the Fluentd-Elasticsearch file from Github
+      unarchive:
+        src: https://github.com/CloudCommandos/missions/raw/CC/orchestrate-containers/k8s%20files/fluentd-elasticsearch.tar
+        dest: /kubernetes
+        remote_src: yes
+
+    - name: Change the default nodeport range
+      lineinfile:
+        path: /etc/kubernetes/manifests/kube-apiserver.yaml
+        insertafter: '    - kube-apiserver'
+        line: '    - --service-node-port-range=80-32767'
+
+    - name: Add the Prometheus url into the ingress file
+      lineinfile:
```

Comment:
You can combine this and the next 3 `lineinfile` tasks into 1 task. Use `with_items` to loop through the changes to be made.